specification/schemas/return-reports/ReturnReport.json

```
@@ -0,0 +1,225 @@
+                      "type": [
+                        "string",
+                        "number"
+                      ],
+                      "description": "Date string in ISO 8601 format or unix timestamp. Only returns approved at >= this date will be in the report.",
+                      "example": "2024-02-01"
+                    },
+                    "date_to": {
+                      "type": [
+                        "string",
+                        "number"
+                      ],
+                      "description": "Date string in ISO 8601 format or unix timestamp. Only returns approved at <= this date will be in the response.",
+                      "example": "2024-03-01"
+                    }
+                  }
+                },
+                "order_reference": {
+                  "type": "string",
+                  "description": "Filter entries by order reference. This is a strict search, meaning that only if the search string matches the `order_reference` attribute of a return completely, will the entries be included.",
+                  "example": "MP000000012"
+                },
+                "item_external_reference": {
+                  "type": "string",
+                  "description": "Filter entries by return order item's reference code in the origin marketplace.",
+                  "example": "#123456"
+                },
+                "item_sku": {
+                  "type": "string",
+                  "description": "Filter entries by return order item's SKU (exact match).",
+                  "example": "SKU123"
+                },
+                "shops": {
+                  "type": "array",
+                  "description": "Array of shop IDs to filter included returns by.",
+                  "items": {
+                    "type": "string",
+                    "format": "uuid",
+                    "pattern": "^[a-f\\d]{8}(-[a-f\\d]{4}){3}-[a-f\\d]{12}$",
+                    "example": "35eddf50-1d84-47a3-8479-6bfda729cd99"
+                  }
+                },
+                "organizations": {
+                  "type": "array",
+                  "description": "Array of organization IDs to filter included returns by.",
+                  "items": {
+                    "type": "string",
+                    "format": "uuid",
+                    "pattern": "^[a-f\\d]{8}(-[a-f\\d]{4}){3}-[a-f\\d]{12}$",
+                    "example": "9cdf86e8-333f-4ed9-bb31-4935c780c947"
+                  }
+                },
+                "brokers": {
+                  "type": "array",
+                  "description": "Array of broker IDs to filter included returns by.",
+                  "items": {
+                    "type": "string",
+                    "format": "uuid",
+                    "pattern": "^[a-f\\d]{8}(-[a-f\\d]{4}){3}-[a-f\\d]{12}$",
+                    "example": "a294ee55-bc94-4890-b734-afd56c158f95"
+                  }
+                },
```

I would re-use the `Uuid` schema and set `example` on the parent if you want to showcase a specific value in the preview JSON, but this is more of a preference, so I'll just leave it as a comment and you can decide what you want to do with it
```suggestion
                "brokers": {
                  "type": "array",
                  "description": "Array of broker IDs to filter included returns by.",
                  "items": {
                    "$ref": "#/components/schemas/Uuid"
                  },
                  "example": ["a294ee55-bc94-4890-b734-afd56c158f95"]
                },
```